microsoft-edge/web-platform/deprecating-ie-support.md

```diff
@@ -0,0 +1,48 @@
+---
+description: Moving users to Microsoft Edge from Internet Explorer 
+title: Moving users to Microsoft Edge from Internet Explorer
+author: MSEdgeTeam
+ms.date: 11/03/2020
+ms.author: msedgedevrel
+ms.prod: microsoft-edge
+keywords: microsoft edge, compatibility, web platform, internet explorer
+---
+# Moving users to Microsoft Edge from Internet Explorer 
+
+Many modern websites have designs that are incompatible with Internet Explorer \(IE\).  Whenever a user of IE visits an incompatible public site, the user may get a message stating the site is incompatible with the browser.  After the message is displayed, the user must manually switch to a different browser.  To minimize disruptions, starting with version 84, Microsoft Edge supports a new capability.  The new capability automatically redirects the user.  When an IE user navigates to a site that is incompatible with IE, Windows automatically redirects the user to Microsoft Edge.  To review the sites on the list, navigate to [Need Microsoft Edge list][MicrosoftEdgeNeededgeV1].
```

Comment:
restate to 

To minimize disruptions, starting with version 84, Microsoft Edge supports a new capability which automatically redirects users.

Reply:
Updated.

```md
To minimize disruptions, starting with version 84, Microsoft Edge supports a new capability that automatically redirects users.
```